### ISSUE TITLE
Can't mount SAML config anymore

### ISSUE THREAD

**Issue by jangrewe** [blackducksoftware/hub#33]
As previously discussed with your support, we're mounting the SAML metadata file from a ConfigMap into the `hub-authentication` and `webapp-logstash` containers, to the path were it would be uploaded when done manually.

This doesn't work anymore with v5.0.0:
```
[webapp-logstash-8f8dcc8db-xnkcr webapp] Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'com.blackducksoftware.usermgmt.sso.config.SAMLMetadataConfig': Unsatisfied dependency expressed through field 'samlContext'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'samlContext': Invocation of init method failed; nested exception is java.nio.file.FileSystemException: /opt/blackduck/hub/hub-webapp/security/SAML_IDP_Metadata.xml: Resource busy
```

**Comment by robpacheco** [comment]
In 5.0.0 I believe we updated this functionality to accept the XML file in the UI instead of having it mounted to the different containers. After the UI upload the different containers that require it manage how they use that XML file.

Have you tried uploading it from the UI, or do you have other needs that require mounting the xml file?

**Comment by robpacheco** [comment]
yes, that config is now stored in the database. There were also some migrations that would be attempted so simplify the process depending on how it was setup originally. Glad it worked!
